classes/Dispatcher.php

```diff
@@ -291,7 +291,10 @@ private function getDefaultController($frontControllerType, Employee $employee =
                 if (null !== $employee) {
                     $tabClassName = $employee->getDefaultTabClassName();
                     if (null !== $tabClassName) {
-                        $defaultController = $tabClassName;
+                        $tabProfileAccess = Profile::getProfileAccess($employee->id_profile, Tab::getIdFromClassName($tabClassName));
+                        if (is_array($tabProfileAccess) && isset($tabProfileAccess['view']) && $tabProfileAccess['view'] == 1) {
```

Comment:
`$tabProfileAccess['view'] == 1`

Consider using strict comparison

Reply:
I used the same comparison as https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/AdminController.php#L562

And the values are string
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Profile.php#L149

I will use strict comparison with `'1'`